src/app/registration/facility-select/facility-select.component.html

```diff
@@ -110,7 +110,7 @@
               Pass availability -
               <span [ngClass]="timeConfig.PM.text">{{ timeConfig.PM.text }}</span>
             </p>
-            <p class="card-text">Arrive after 12pm</p>
+            <p class="card-text">Arrive after 1pm</p>
```

Comment:
Our coverage on this file is a little weak, and it doesn't look like we actually test this area.  Would be cool if we could throw in a quick UT on this one.